brainframe/cli/docker_compose.py

```diff
@@ -98,3 +104,44 @@ def check_existing_version(install_path: Path):
     version = compose["services"]["core"]["image"].split(":")[-1]
     version = "v" + version
     return version
+
+
+def _assert_has_docker_permissions():
+    """Fails if the user does not have permissions to interact with Docker"""
+    if not (os_utils.is_root() or os_utils.currently_in_group("docker")):
+        error_message = (
+            i18n.t("general.docker-bad-permissions")
+            + "\n"
+            + _group_recommendation_message("docker")
+        )
+
+        print_utils.fail(error_message)
+
+
+def _assert_has_write_permissions(path: Path):
+    """Fails if the user does not have write access to the given path."""
+    if os.access(path, os.W_OK):
+        return
+
+    error_message = i18n.t("general.file-bad-write-permissions", path=path)
+
+    print()
```

Comment:
Nit: this print feels kinda strange hanging out here alone? Could/should it be combined with the error, or at least moved closer?